### ISSUE TITLE
Column names to allow for nested Objects fieldtype

### ISSUE THREAD

**Issue by harinij** [rowyio/rowy#161]
Currently Firetable allows for only camelcase field naming, expand it to allow for nested objects retrieval

**Comment by AntlerEngineering** [comment]
implemented in #165

**Comment by n-sviridenko** [comment]
Single select writes the data in both the old and the new ways. For example, if the field name is `project.name`, in the db it'll look like:

```js
{
  'project.name': 'Test', // <-- this is redundant
  project: {
    name: 'Test'
  }
}
```

**Comment by murat0** [comment]
> Single select writes the data in both the old and the new ways. For example, if the field name is `project.name`, in the db it'll look like:
> 
> ```js
> {
>   'project.name': 'Test', // <-- this is redundant
>   project: {
>     name: 'Test'
>   }
> }
> ```

Not just single select - when I try and add a nested field with a type using "parent.child" notation I get duplicate entries in firestore.

**Comment by notsidney** [comment]
Hi, this is fixed as of [v2.0.0](https://github.com/rowyio/rowy/releases/v2.0.0)

You can migrate the latest version of Rowy by using this easy deploy link on your Firestore project: [rowy.app/deploy](rowy.app/deploy). Try it out and we would love to get your feedback [on Discord](https://discord.gg/B8yAD5PDX4) 🙏
